contracts/BaseStrategy.sol

```diff
@@ -149,6 +153,21 @@ abstract contract BaseStrategy {
      */
     function name() external virtual pure returns (string memory);
 
+    /**
+     * @notice
+     *  The amount (priced in want) of the total assets managed by this strategy should not count
+     *  towards Yearn's TVL calculations.
+     * @dev
+     *  You can override this field to set it to a non-zero value if some of the assets of this
+     *  Strategy is somehow delegated inside another part of of Yearn's ecosystem e.g. another Vault.
+     *  Note that this value must be strictly less than or equal to the amount provided by
+     *  `estimatedTotalAssets()` below, as the TVL calc will be total assets minus delegated assets.
```

Comment:
Not clear on the comment, is it correct to assume estimated total assets should be a sum of total assets in control of the strat + delegated assets? If this is the case we can make this non virtual and make the two sub method virtual? 

```suggestion
     *  EstimatedTotalAssets= 
TotalAssets() + delegatedAssets()
```

Reply:
No, `delegatedAssets()` has a particular meaning for TVL calculations only. It means that amount is delegated to another Yearn component accounted for in TVL calculations, and as such should be subtracted from `estimatedTotalAssets()` here in order to conduct the calculation.

Reply:
Makes sense, thanks for explanation